lib/libzfs/libzfs_sendrecv.c

```diff
@@ -5390,45 +5390,21 @@ zfs_receive(libzfs_handle_t *hdl, const char *tosnap, nvlist_t *props,
 		return (-2);
 	}
 
-#ifdef __linux__
-#ifndef F_SETPIPE_SZ
-#define	F_SETPIPE_SZ (F_SETLEASE + 7)
-#endif /* F_SETPIPE_SZ */
-
-#ifndef F_GETPIPE_SZ
-#define	F_GETPIPE_SZ (F_GETLEASE + 7)
-#endif /* F_GETPIPE_SZ */
-
 	/*
 	 * It is not uncommon for gigabytes to be processed in zfs receive.
 	 * Speculatively increase the buffer size via Linux-specific fcntl()
 	 * call.
```

Comment:
How about "Speculatively increase the buffer size if supported by the platform".